library/ajax/imm_autocomplete/search.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/** Copyright (C) 2016 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @link    http://www.open-emr.org
+ */
+
+//SANITIZE ALL ESCAPES
+$sanitize_all_escapes=true;
+//
+
+//STOP FAKE REGISTER GLOBALS
+$fake_register_globals=false;
+//
+require_once('../../../interface/globals.php');
+
+
+if (isset($_GET['term'])){
+    $term = $_GET['term'];
+	$return_arr = array();
+
+	try {
+		$sql = "SELECT DISTINCT lot_number FROM immunizations WHERE lot_number LIKE ? ";
+
+		$res = sqlstatement($sql,"%".$term."%");
+		while($row = sqlFetchArray($res)){
+			$return_arr[] =  $row['lot_number'] ;
+		}
+
+	
+	} catch(PDOException $e) {
```

Comment:
It started out using PDO and I had to convert it and forgot to remove the last remaining remnants of PDO.